src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,7 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService use = new UserService();
+        return use.findByEmail(email) != null && (use.findByEmail(email).getPassword().equals(password));
```

Comment:
if `findByEmail` takes some time, this method will perform better if we introduce a variable for return object of `findByEmail` method